RELEASE-NOTES.txt

```diff
@@ -1,3 +1,24 @@
+1.3.3
+=====
+* Handle zip imports regression #874
+* Sentry: change release naming #868
+* Sentry: improve breadcrumbs with providing more info #864
+* Sentry: add user identifier #863
```

Comment:
This isn't user-facing. My instinct would be not to include it. Thoughts, @sejas?

Reply:
I was thinking about it, but noticed that Sentry notes were added before, so decide that such type of changes probably should be included.
@sejas WDYT, should we include or it's better to keep only user facing changes?

Reply:
Yeah, Let's remove what's not relevant to users and are small changes.
We can join all the sentry entries in a single line. Like : `Improved Sentry reports #863, #864....`